export/platform/gc_design_system/airtable/main.py

```diff
@@ -0,0 +1,87 @@
+import json
+import boto3
+import os
+import requests
+
+# Environment variables
+AIRTABLE_API_KEY_PARAMETER_NAME = os.environ.get("AIRTABLE_API_KEY_PARAMETER_NAME")
+S3_BUCKET_NAME = os.environ.get("S3_BUCKET_NAME")
+S3_OBJECT_PREFIX = os.environ.get("S3_OBJECT_PREFIX")
+
+# Airtable configuration (hardcoded like Freshdesk domain)
+AIRTABLE_BASE_ID = "appaMppKljeU8zJE1"
+AIRTABLE_TABLE_NAME = "tbllQrJwtozOb0Ziv"
```

Comment:
Could you switch these to environment variables that are passed into the function